flareon-macros/src/lib.rs

```diff
@@ -112,6 +114,14 @@ pub fn query(input: TokenStream) -> TokenStream {
     query_to_tokens(query_input).into()
 }
 
+#[proc_macro_attribute]
+pub fn dbtest(_args: TokenStream, input: TokenStream) -> TokenStream {
```

Comment:
Why pub?

Reply:
It is defined in a different crate, so it needs to be pub to be accessible. It's not reexported in `flareon`.